src/components/twitchChat.vue

```diff
@@ -1,26 +1,23 @@
 <script setup lang="ts">
+import { ref } from 'vue';
 
+const host = ref(window.location.hostname);
 </script>
 
 <template>
-    <div class="twitch-iframe" style="flex-direction: column;">
-        <h2>Ask for help, or try it out!</h2>
-        
-        <iframe id="twitch-chat-embed"
-          src="https://www.twitch.tv/embed/potatbotat/chat?parent=potat.app&darkpopout"
-          height="500"
-          width="350"
-          style="border-radius: 15px; overflow: hidden;">
-        </iframe>
-    </div>
+	<div class="twitch-iframe">
+		<h2>Ask for help, or try it out!</h2>
+
+		<iframe id="twitch-chat-embed"
+    	:src="`https://www.twitch.tv/embed/potatbotat/chat?parent=${host}&darkpopout`"
+			height="550"
+			width="100%"
```

Comment:
pls add back 'style="border-radius: 15px;">' inline or in the twitch-chat-embed id

Reply:
ok it's back, it's deadass ugly tho (atleast for me).

Reply:
also text-align: center; for the twitch-iframe pls